dev/bots/analyze.dart

```diff
@@ -383,8 +383,8 @@ bool _matches<T>(List<T> a, List<T> b) {
   return true;
 }
 
-final RegExp _importPattern = RegExp(r"import 'package:flutter/([^.]+)\.dart'");
-final RegExp _importMetaPattern = RegExp(r"import 'package:meta/meta.dart'");
+final RegExp _importPattern = RegExp(r'''^\s*import (['"])package:flutter/([^.]+)\.dart\1''');
+final RegExp _importMetaPattern = RegExp(r'''^\s*import (['"])package:meta/meta.dart\1''');
```

Comment:
How do these ignore comments?

Reply:
The "^" anchors the search at the start of the line.

Reply:
mind = blown